src/pages/Login.js

```diff
@@ -3,17 +3,25 @@ import LoginForm from "../components/login/login_form"
 
 class Login extends React.Component{
     state = {
-        loginError : false
+        loginError : false,
+        errMessage: ""
     }
     
+    //change the state of error in login
```

Comment:
Comments usually add complexity to code. In the future, if you have to make a change in the code, you'll have to maintain the code and the comments. Most of the times code can be self explanatory.

Reply:
ok got it! but, if I am implementing something that my team is not aware of or don´t know about a new syntax or functionality, how should I handle that?